.github/workflows/cleanup-clusters.yml

```diff
@@ -1,21 +1,23 @@
-name: Wipe all clusters and apps on dev
+name: Wipe all clusters and apps
 
 on:
+  schedule:
+    - cron: "0 6 * * *"
```

Comment:
I think it is unlike anyone from the team is working at that point but maybe we should do the clean up a bit earlier to be a bit more safe 😴

Reply:
I have updated it for 2am 😪